easier68k/core/opcodes/Bcc.py

```diff
@@ -364,4 +364,403 @@ def disassemble_instruction(cls, data: bytes):
+        return bcc_disassemble_instruction(data, BVS, 9)
+
+    @classmethod
+    def from_str(cls, command: str, parameters: str):
+        return bcc_from_str(command, parameters, BVS)
+
+
+class BPL(branch_code):
+    """
+    The BPL opcode compiles down to 6Axx xxxx xxxx
+    The conditional here is NOT N
+    """
+
+    cond_code = '\x0A'
+
+    def conditional(self, simulator: M68K):
+        n = simulator.get_condition_status_code(ConditionStatusCode.N)
+
+        if not n:
+            return True
+
+        return False
+
+    @classmethod
+    def command_matches(cls, command: str):
+        return opcode_util.command_matches(command, 'BPL')
+
+    @classmethod
+    def disassemble_instruction(cls, data: bytes):
+        return bcc_disassemble_instruction(data, BPL, ord('\x0A'))
+
+    @classmethod
+    def from_str(cls, command: str, parameters: str):
+        return bcc_from_str(command, parameters, BPL)
+
+
+class BMI(branch_code):
+    """
+    The BMI opcode compiles down to 6Bxx xxxx xxxx
+    The conditional here is N
+    """
+
+    cond_code = '\x0B'
+
+    def conditional(self, simulator: M68K):
+        n = simulator.get_condition_status_code(ConditionStatusCode.N)
+
+        if n:
+            return True
+
+        return False
+
+    @classmethod
+    def command_matches(cls, command: str):
+        return opcode_util.command_matches(command, 'BMI')
+
+    @classmethod
+    def disassemble_instruction(cls, data: bytes):
+        return bcc_disassemble_instruction(data, BMI, ord('\x0B'))
+
+    @classmethod
+    def from_str(cls, command: str, parameters: str):
+        return bcc_from_str(command, parameters, BMI)
+
+
+class BGE(branch_code):
+    """
+    The BGE opcode compiles down to 6Cxx xxxx xxxx
+    The conditional here is N AND V OR NOT N AND NOT V
+    """
+
+    cond_code = '\x0C'
+
+    def conditional(self, simulator: M68K):
+        n = simulator.get_condition_status_code(ConditionStatusCode.N)
+        v = simulator.get_condition_status_code(ConditionStatusCode.V)
+
+        if n and v:
+            return True
+
+        if not n and not v:
+            return True
+
+        return False
+
+    @classmethod
+    def command_matches(cls, command: str):
+        return opcode_util.command_matches(command, 'BGE')
+
+    @classmethod
+    def disassemble_instruction(cls, data: bytes):
+        return bcc_disassemble_instruction(data, BGE, ord('\x0C'))
+
+    @classmethod
+    def from_str(cls, command: str, parameters: str):
+        return bcc_from_str(command, parameters, BGE)
+
+
+class BLT(branch_code):
+    """
+    The BLT opcode compiles down to 6Dxx xxxx xxxx
+    The conditional here is N AND NOT V OR NOT N AND V
+    """
+
+    cond_code = '\x0D'
+
+    def conditional(self, simulator: M68K):
+        n = simulator.get_condition_status_code(ConditionStatusCode.N)
+        v = simulator.get_condition_status_code(ConditionStatusCode.V)
+
+        if n and v:
```

Comment:
This does not match `N AND NOT V OR NOT N AND V`
Also, we could probably do `return n != v`

Reply:
broke the tests so I assume not